fv/test-connection/test-connection.go

```diff
@@ -119,7 +140,16 @@ func main() {
 	}
 }
 
-func tryConnect(ipAddress, port, sourcePort, protocol, loopFile string) error {
+func maybeAddInterface(sourceIP string) error {
```

Comment:
this function is poorly named, it should be `maybeAddAddr` (or `maybeAddAddrToInterface` if you want)

Reply:
46acbf95f898593f230a8cb4762973055aecb9af